src/pychart/_interpreter/expression.py

```diff
@@ -142,6 +142,30 @@ def __call__(self, environment: Environment):
         return environment.mutate(self.name.lexeme, self.initializer(environment))
 
 
+class Call(Expr):
+    callee: Expr
+    arguments: List[Expr]
+
```

Comment:
`Call` expression node which is generated to match stuff like `funcy()` has 2 fields, the callee (the name of the function // Variable expression that retrieves function definition callable) and the args (can be any valid return type)